src/modules/admin/components/users/UserAuditPage.tsx

```diff
@@ -36,9 +52,13 @@ const toggleItems: ToggleItem<EntityType>[] = [
 ];
 
 interface Props {
-  entityType: EntityType;
+  pageEntityType: PageEntityType;
+  accessEntityType?: AccessEntityType;
 }
-const UserAuditPage: React.FC<Props> = ({ entityType }) => {
+const UserAuditPage: React.FC<Props> = ({
```

Comment:
This component is feeling pretty bloated, as it is responsible for the state of both toggles.

I would recommend creating a new component `UserAccessHistory` that renders this (red) part of the page
<img width="1312" alt="Screenshot 2024-02-07 at 3 40 11 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/f9fdeae9-2989-49e6-ba5c-1518276e80f5">


Then you would have a separation of concerns like this:
* UserAuditPage: renders the page title and toggle, owns the Audit History Type toggle state
* UserAuditHistory: renders "User Edits" card content
* UserAccessHistory: renders "User Access" card content. Owns the Client-vs-Enrollment toggle state.

Reply:
Oh good thinking. Thanks 🙏